assets/dev/scss/frontend/widgets/divider.scss

```diff
@@ -12,6 +12,7 @@
 	--divider-pattern-size: 20px;
 	--divider-pattern-url: none;
 	--divider-pattern-repeat: repeat-x;
+  	--divider-default-width:100%;
```

Comment:
This variable might be redundant. I believe Louis has added a default value for `--container-widget-width` in his gallery task already.

Reply:
Done